src/descriptions.js

```diff
@@ -15,99 +15,99 @@ module.exports = {
+  "sect leader": "Each night, you can convert a player to join your sect by doing `+convert <player>`. Converted players keep their original role. All sect members will flee if the Sect Leader dies. You cannot convert werewolves and solo killers. Your aura is **Unknown** and you belong to the **Sect** team.",
+  zombie: "Every night you can bite a player by doing `+convert <player>` which will turn that player into a zombie at the end of the next night. Bitten players remain on their team until they convert, and the converting process will be stopped if the zombie that bit them dies before they are turned. Each zombie can select a different target. Werewolves and solo killers cannot be converted. Zombies, other than the original will decay and die after two nights as a zombie or if they attempt to bite a werewolf. They will also die if they attempt to bite a werewolf. Your aura is **Unknown** and you belong to the **Zombie** team.",
+  alchemist: "You have two potions, red and black. Every night, you can give them to two different players by doing `+give <player> <red | black>`. Both of them will be warned that they might die at the end of the day but only the player with the black potion will actually die. The red potion is only deadly the second time it is given. They don't know which potion they have. Your aura is **Unknown** and you are a **Solo** team player.",
+  "evil detective": "Each night you can select two players by doing `+check <player 1> <player 2>`, including yourself. If the two players are in different teams, then they will both die, otherwise nothing happens. If you select yourself, only the other player dies. Your aura is **Unknown** and you are a **Solo** team player.",
 // CUSTOM ROLES
   modded: "You are a modded role. When you receive this role, you will get further instructions from the narrator on what you are supposed to do.",
 // CUSTOM VILLAGE ROLES
-  undertaker: "After Day 1, you may select a player using `+watch` to watch them. If that player is killed, you can know one of their teammates at the end of that game phase. You may only select one player per game. Your aura is **Good** and you are a **Village** team player.",
-  prognosticator: "At night, you can announce that no one will die the next night by using `+peace`. At day, you can announce that the chosen player's vote will be taken away the next day by using `+terror`. Your aura is **Unknown** and you are a **Village** team player.",
+  undertaker: "After the first day, you may select a player using `+watch` to watch them. If that player is killed, you can know one of their teammates at the end of that game phase. You may only select one player per game. Your aura is **Good** and you belong to the **Village** team.",
```

Comment:
```suggestion
  undertaker: "After the first day, you may select a player using `+watch <player>` to watch them. If that player is killed, you can know one of their teammates at the end of that game phase. You may only select one player per game. Your aura is **Good** and you belong to the **Village** team.",
```
I think the command has an arg. Correct me if not